style.css

```diff
@@ -0,0 +1,41 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background-color: hsl(41, 70%, 94%);
+  color: hsl(41, 70%, 20%);
+  font-family: "Libre Baskerville", serif;
+}
+
+h1 {
+  background-color: white;
+  color: rgb(31, 70, 122);
+  font-size: 40px;
+  text-align: center;
+  text-transform: uppercase;
+}
+
+h2 {
+  font-size: 22px;
+  line-height: 50px;
+  text-align: center;
+}
+
+
+p {
+  font-size: 16px;
+  text-align: center;
+}
+
+strong {
+  text-decoration: underline;
+  font-weight: bold;
+}
+
+em {
+  color: rgb(31, 70, 122);
+  font-family: "Courgette", "Libre Baskerville", cursive;
+  font-size: 18px;
+  font-style: italic;
+}
```

Comment:
Overall, great work -- most of my comments on your CSS here are refinements, there are no errors. ⭐